package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Technically this is a bugfix not a new feature. So I think `2.0.1` would be more appropriate

Reply:
fixed